### PR TITLE
 Adding tinydrive starters to the nav

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: N/A

Description of Changes:
* adding "missing" tiny drive quickstarts to the docs
* fixing tiny drive quickstarts index page
* merging script change into master branch

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _scripts/icon_list_refresh.sh [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+  echo "Provide the path to the TinyMCE source root directory"
+  exit 1
+else
+  tinymce_root=$1
+fi
 # Output markdown file
 list_file='_includes/configuration/icon_list.md'
 # Input directory containing the icon SVGs
 icon_svg_dir='images/icons/'
+rm -rf $icon_svg_dir*
```
Comment (ashfordneil) [inline _scripts/icon_list_refresh.sh:14]: There's a check just below this in the script to see whether we're running the script in the wrong directory. Should this section be moved to after that check, in case we accidentally `rm -rf` the wrong thing? (Also, would `rm -r` work? I'm not a big fan of squashing the errors in a destructive command if we can avoid it).

File: _scripts/icon_list_refresh.sh [hunk 2]
```diff
@@ -1,9 +1,18 @@
 #!/bin/bash
 
+if [ -z $1 ]
```
Comment (ashfordneil) [inline _scripts/icon_list_refresh.sh:3]: Happy to review the script, but is this related to adding tinydrive starters to the nav or should it be in a separate PR?

Comment (tylerkelly13) [reply _scripts/icon_list_refresh.sh:3]: Hi Neil,

This script was added by PR https://github.com/tinymce/tinymce-docs/pull/1618, I accidentally created the PR on the develop branch, not master. This script is for generating the icon list in the docs, it's destructive because if an icon is removed, we need to remove it from the docs (and might also allow us to see when one is accidentally removed from or changed in the TinyMCE repo).
